File: app/src/main/java/com/androgynousis/messenger/view/holder/HolderDate.java
package com.androgynousis.messenger.view.holder;

import androidx.recyclerview.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.androgynousis.messenger.R;

public class HolderDate extends RecyclerView.ViewHolder {

    private TextView date;

    public HolderDate(View v) {
        super(v);
        date = (TextView) v.findViewById(R.id.tv_date);
    }

    public TextView getDate() {
        return date;
    }

    public void setDate(TextView date) {
        this.date = date;
    }
}

File: app/src/main/java/com/androgynousis/messenger/view/holder/HolderDate.kt
package com.androgynousis.messenger.view.holder

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.androgynousis.messenger.R

class HolderDate(v: View) : RecyclerView.ViewHolder(v) {
    var date: TextView = v.findViewById<View>(R.id.tv_date) as TextView
}